meta/heimseiten/contao-tiny-slider-bundle/de.yml

```
@@ -0,0 +1,8 @@
+de:
+    title: Tiny Slider
+    description: >
+        Ersetzt den standard Contao-Slider mit dem Tiny Slider und fügt Funktionen hinzu wie Fade, Ken-Burns-Effekt usw..
```

```suggestion
        Ersetzt den Standard Contao-Slider mit dem Tiny Slider und fügt Funktionen hinzu wie Fade, Ken-Burns-Effekt usw..
```

There is no adjective called "standard" in German, is there? Afaik you always capitalise it, when using it this way.